inputfiles/overridingTypes.jsonc

```diff
@@ -2792,6 +2792,16 @@
                         }
                     }
                 }
+            },
+            "HTMLMediaElement": {
+                "properties": {
+                    "property": {
+                        "preload": {
+                            "name": "preload",
```

Comment:
```suggestion
```

"name" is not needed in this case.

Reply:
Is it not? Can you elaborate when it is needed and when not? I've seen it plenty of times in the code. Or is the usage just deprecated (if name matches) and these are leftovers?